src/reducers.js

```diff
@@ -32,6 +32,22 @@ const reducers = (state = {}, action) => {
         ...state,
         isLoading,
       };
+    case ActionType.EDIT_PLAYER:
+      const { player } = action
+      return {
+        ...state,
+        editingPlayer: player
+      }
+    case ActionType.SAVE_PLAYER:
```

Comment:
Recommend renaming `EDIT_PLAYER` and `SAVE_PLAYER` to `SET_EDITING_PLAYER` and `RESET_EDITING_PLAYER`.